viam-cartographer/src/slam_service/slam_service.cc

```diff
@@ -396,15 +398,9 @@ void SLAMServiceImpl::GetLatestSampledPointCloudMapString(
             float z_pos = 0;  // Z is 0 in 2D SLAM
 
             // Add point to buffer
-            Eigen::Vector3d map_point(x_pos, y_pos, z_pos);
-            Eigen::Vector3d rotated_map_point = pcdRotation * map_point;
-
-            viam::utils::writeFloatToBufferInBytes(pcd_data,
-                                                   rotated_map_point.x());
-            viam::utils::writeFloatToBufferInBytes(pcd_data,
-                                                   rotated_map_point.y());
-            viam::utils::writeFloatToBufferInBytes(pcd_data,
-                                                   rotated_map_point.z());
+            viam::utils::writeFloatToBufferInBytes(pcd_data, x_pos);
```

Comment:
Why are we still flipping the sign of y_pos?

Reply:
I have chosen not to deal with the flipping issue in this PR as I cannot validate the quaternion without the SLAM UI update or some substantial work messing around with quaternions. Therefore the negative y_pos was left unaffected by this PR

Reply:
got it, sounds good